src/iris/bin/sender.py

```diff
@@ -1441,6 +1441,18 @@ def sender_shutdown():
     os._exit(0)
 
 
+def check_call_restrictions(message):
+    # check for known corner cases in message delivery and correct accordingly
+
+    # due to calling restrictions to china override mode and send as sms instead
+    if message['destination'].startswith('+86'):
+        message['mode'] = 'sms'
```

Comment:
We update message twice, once in the helper function and once in `message_send_enqueue`. Not sure there's enough code to bother with a helper function if we move the edits out, but `check_call_restrictions` is a misleading name for a function with side effects. Recommend just putting all this in message_send_enqueue.

Reply:
Fixed to only update message once. Also I pulled this functionality out into a helper function so other calling restrictions (like certain India calling restrictions) could be added in the future.